website/src/models/group.js

```diff
@@ -168,6 +170,64 @@ schema.methods.isMember = function isGroupMember (user) {
   }
 };
 
+schema.methods.startQuest = function startQuest () {
+  if (this.type !== 'party') throw new BadRequest('Must be a party to use this method');
+  if (!this.quest.key) throw new BadRequest('Party does not have a pending quest');
+  if (this.quest.active) throw new BadRequest('Quest is already active');
```

Comment:
I think it's going to have to be asyncronous, actually, because it needs to fetch the email details from the members.